apps/system/mem_leak_checker/README.md

```diff
@@ -0,0 +1,67 @@
+# How to use Memory Leakage Checker
+
+The Memory Leakage Checker shows leakage information of dynamic allocated memory at that time.  
+It checks whole RAM regions with allocated memory address. If no region has that address,  
```

Comment:
Please change "with" -> "for"
"checks entire RAM regions for an allocated memory address"